### ISSUE TITLE
quickstart dev steps in README do not document test dependency upon SYSL_PLANTUML

### ISSUE THREAD

**Issue by fcostin** [anz-bank/sysl#148]
### Expected behaviour

after git cloning sysl, following steps in https://github.com/anz-bank/sysl#development should suffice to run tests locally in a dev environment

### Actual behaviour

```
(.venv) rfc@rfc-t410:~/projects/sysl$ pytest
================================================================== test session starts ==================================================================
platform linux2 -- Python 2.7.12, pytest-3.6.2, py-1.5.4, pluggy-0.6.0
rootdir: /home/rfc/projects/sysl, inifile:
collected 37 items                                                                                                                                      

test/e2e/test_e2e.py ........F                                                                                                                    [ 24%]
test/exporters/reljam_test.py ...                                                                                                                 [ 32%]
test/exporters/xsd_test.py .....                                                                                                                  [ 45%]
test/sysl/test_annotations.py ......                                                                                                              [ 62%]
test/sysl/test_epa.py FFFFFFFFFFF                                                                                                                 [ 91%]
test/sysl/test_sysldata.py .F                                                                                                                     [ 97%]
test/util/algo_test.py .                                                                                                                          [100%]

======================================================================= FAILURES ========================================================================

[...etc...]

E           ConnectionError: HTTPConnectionPool(host='localhost', port=8080): Max retries exceeded with url: /plantuml/img/ZP112u9058Jl-oicBrjhYdeC4QLTeeiBiPr3L50m2tSxHFJVsooaI-3StcE-USEZTAu8fPYbhpKxAhaJkJXaId1i9HVmF729N2e8lbUBoJexTNPNacfJ3EPsxaZP5Lh3KVTh7JeeD4u1uXXBlhuCPTDMAoG97iI52TiA6xIz8SzVAlfGuHygcYWNJQ77EtR1cAuDud5C6BEHDW7Cli09V9yvjWP9wxvwLtm1 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f842ff2de10>: Failed to establish a new connection: [Errno 111] Connection refused',))

test/sysl/test_sysldata.py:36: AssertionError
========================================================= 13 failed, 24 passed in 1.69 seconds ==========================================================

```
### Steps to reproduce the behaviour

start with your favourite linux environment that has git, python27 and virtualenv installed:

```
git clone https://github.com/anz-bank/sysl.git
cd sysl
virtualenv .venv
source .venv/bin/activate
pip install pytest flake8 -e .
pytest
```

environment details:

* `python --version` : `Python 2.7.12`
* `virtualenv --version` :  `15.0.1`
* `cat /proc/version` :  `Linux version 4.4.0-127-generic (buildd@lcy01-amd64-023) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.9) ) #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018`
* `git log -n 1 --format=oneline HEAD` :  `2ad3c50f42386488d0a67c072179fcbe07eed53c Fix trailing whitespace handling for rest apis (#147)`

It's reasonably easy to figure out what's going on by staring at the tracebacks, then observing that Travis claims the build is passing, and going to read the `.travis.yml`. But it'd be much friendlier if this `SYSL_PLANTUML` dependency was documented --- perhaps ideally the test suite could provision its own PLANTUML endpoint running on localhost . I see the Travis CI test suite will fail if a certain external service named `http://www.plantuml.com/` happens to go down.

**Comment by juliaogris** [comment]
PLANTUML is a Java app running in Tomcat depending on graphviz I believe, so not super trivial to setup locally (by tests).
I'll add add to the README to do one of the following 
* execute `SYSL_PLANTUML=http://www.plantuml.com/plantuml`
* add `export SYSL_PLANTUML=http://www.plantuml.com/plantuml` to .bashrc or similar
* install PLANTUML locally and run on port 8080
with a warning that for sensitive data the public service at `www.plantuml.com` is hot suitable.

Happy?

**Comment by juliaogris** [comment]
Fixed by #151 -  happy to close this @anz-rfc ?

**Comment by anz-rfc** [comment]
thanks @juliaogris
